src/components/apiDelegation/given/ReceiptPage/ReceiptPage.tsx

```diff
@@ -15,14 +15,15 @@ import { resetDelegableOrgs } from '@/rtk/features/delegableOrg/delegableOrgSlic
 import { resetDelegableApis } from '@/rtk/features/delegableApi/delegableApiSlice';
 import { RouterPath } from '@/routes/Router';
 import { ReactComponent as ApiIcon } from '@/assets/ShakeHands.svg';
+import { setLoading as setOveviewToReload } from '@/rtk/features/overviewOrg/overviewOrgSlice';
```

Comment:
Delete the 'as setOveviewToReload', there's no other setLoading here.
```suggestion
import { setLoading } from '@/rtk/features/overviewOrg/overviewOrgSlice';
```

Reply:
Yeah, but I figured it would be more readable like this, since the loading isn't actually for this page, but for another page, which this name makes clear.